backend/pkg/config/ratelimits/config.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2025 Northern.tech AS
+//
+//	Licensed under the Apache License, Version 2.0 (the "License");
+//	you may not use this file except in compliance with the License.
+//	You may obtain a copy of the License at
+//
+//	    http://www.apache.org/licenses/LICENSE-2.0
+//
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+
+package ratelimits
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mendersoftware/mender-server/pkg/config"
+	"github.com/mendersoftware/mender-server/pkg/log"
+	"github.com/mendersoftware/mender-server/pkg/rate"
+	"github.com/mendersoftware/mender-server/pkg/redis"
+)
+
+func init() {
+	config.Config.SetDefault(SettingRatelimitsAuthDefaultInterval, "1m")
+	config.Config.SetDefault(SettingRatelimitsAuthDefaultQuota, "120")
+	config.Config.SetDefault(SettingRatelimitsAuthDefaultEventExpression,
+		"{{with .Identity}}{{.Subject}}{{end}}")
+}
+
+const (
+	SettingRatelimits                           = "ratelimits"
+	SettingRatelimitsAuth                       = SettingRatelimits + ".auth"
+	SettingRatelimitsAuthEnable                 = SettingRatelimitsAuth + ".enable"
+	SettingRatelimitsAuthGroups                 = SettingRatelimitsAuth + ".groups"
+	SettingRatelimitsAuthMatch                  = SettingRatelimitsAuth + ".match"
+	SettingRatelimitsAuthDefault                = SettingRatelimitsAuth + ".default"
+	SettingRatelimitsAuthDefaultQuota           = SettingRatelimitsAuthDefault + ".quota"
+	SettingRatelimitsAuthDefaultInterval        = SettingRatelimitsAuthDefault + ".interval"
+	SettingRatelimitsAuthDefaultEventExpression = SettingRatelimitsAuthDefault +
+		".event_expression"
+)
+
+func LoadRatelimits(c config.Reader) (*RatelimitConfig, error) {
+	if !c.GetBool(SettingRatelimitsAuthEnable) {
+		return nil, nil
```

Comment:
just to double check, nil,nil here, ErrNotEnabled would be easier on the caller.